src/commands/synthetics/run-tests-command.ts

```diff
@@ -299,6 +301,7 @@ export class RunTestsCommand extends Command {
         startUrl: process.env.DATADOG_SYNTHETICS_OVERRIDE_START_URL,
         startUrlSubstitutionRegex: process.env.DATADOG_SYNTHETICS_OVERRIDE_START_URL_SUBSTITUTION_REGEX,
         testTimeout: toNumber(process.env.DATADOG_SYNTHETICS_OVERRIDE_TEST_TIMEOUT),
+        variables: toStringObject(process.env.DATADOG_SYNTHETICS_OVERRIDE_VARIABLES),
```

Comment:
Can you please add a test case for the env variable. I think this one is the correct test - https://github.com/DataDog/datadog-ci/pull/1354/files#diff-fed723dc96fbca46fa0fc4add02e5a092a935c4778780668acc07e993bc916deR60